src/probnum/randprocs/markov/continuous/_sde.py

```diff
@@ -1,38 +1,58 @@
 """SDE models as transitions."""
-import functools
+
 from typing import Callable, Optional
 
 import numpy as np
-import scipy.integrate
-import scipy.linalg
 
-from probnum import randvars
-from probnum.randprocs.markov import _transition, discrete
-from probnum.randprocs.markov.continuous import _mfd
+from probnum.randprocs.markov import _transition
 from probnum.typing import FloatArgType, IntArgType
-from probnum.utils.linalg import tril_to_positive_tril
 
 
 class SDE(_transition.Transition):
-    """Stochastic differential equation.
+    r"""Stochastic differential equation.
 
-    .. math:: d x(t) = g(t, x(t)) d t + L(t) d w(t),
+    .. math:: d x(t) = g(t, x(t)) d t + l(t, x(t)) d w(t),
 
-    driven by a Wiener process with unit diffusion.
+    driven by a Wiener process :math:`w` with isotropic diffusion :math:`\Gamma(t) = \gamma(t) I_d`.
     """
 
     def __init__(
         self,
-        dimension: IntArgType,
-        driftfun: Callable[[FloatArgType, np.ndarray], np.ndarray],
-        dispmatfun: Callable[[FloatArgType, np.ndarray], np.ndarray],
-        jacobfun: Callable[[FloatArgType, np.ndarray], np.ndarray],
+        state_dimension: IntArgType,
+        wiener_process_dimension: IntArgType,
+        drift_function: Callable[[FloatArgType, np.ndarray], np.ndarray],
+        dispersion_function: Callable[[FloatArgType, np.ndarray], np.ndarray],
```

Comment:
dispersion functions are now (t, x) dependent. This is a lack of generality that was bugging me before.